html/index.html

```diff
@@ -36,21 +36,22 @@ <h5>Existing samples</h5>
         </tr>
         {% endfor %}
       </table>
-<!-- 
       <h5>Missing high-statistics samples with number of events bigger than 20M </h5>
       <table>
         <tr>
           <th>Campaign group</th>
         </tr>
-        {% for campaign_group in campaign_groups %}
-          <tr>
-            <td>
-              <a href="missing_group/{{campaign_group}}">{{campaign_group}}</a>
-            </td>
-          </tr>
-        {% endfor %}
+        <tr>
+          <td>
+            <a href="missing_page/RunIISummer19UL18">RunIISummer19UL18</a>
```

Comment:
It would be better to avoid hard-coding campaign names. You can fetch them from the database in `index()`, just like it is done for "existing samples" table above.